cmd/substreams-sink-postgres/inject_cursor.go

```diff
@@ -0,0 +1,145 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+	"time"
+
+	_ "github.com/lib/pq"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	"github.com/streamingfast/cli"
+	. "github.com/streamingfast/cli"
+	"github.com/streamingfast/cli/sflags"
+	"github.com/streamingfast/dstore"
+	sink "github.com/streamingfast/substreams-sink"
+
+	"github.com/streamingfast/substreams-sink-postgres/db"
+	"github.com/streamingfast/substreams-sink-postgres/state"
+	"go.uber.org/zap"
+)
+
+var injectCursorCmd = Command(injectCursor,
+	"inject-cursor <input-path> <psql-dsn> <endpoint> <manifest> <outputModuleName>",
```

Comment:
Could we merge that step straight into the `inject-csv` command, so it would kind of be the last step of the single command you need to insert + set cursor?

Reply:
I was planning to do this, but I thought that in case there are multiples tables, it would insert multiple times the same cursor

Reply:
Actually, I got an issue where I was injecting the csv and for some reason it shut down and started from the beginning.  Maybe we could have some kind of "cursor" for each table for `inject-csv`.

Reply:
> I was planning to do this, but I thought that in case there are multiples tables, it would insert multiple times the same cursor

You are right, we would need indeed to write the good cursors only at the very end once all tables have successfully been copied. We could indeed have a cursor for each table and write the final cursor once the final has been written (is it possible to "know" we are the last table?).

Anyway, all this said, I'm good to merge it in its current form and expand upon it later.